apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -554,24 +571,40 @@ const ExportContent = () => {
             Step 3
           </Text>
           <Text color="subtle">
-            Use the{" "}
+            Run this command to publish to{" "}
             <StyledLink
-              href="https://vercel.com/docs/cli"
+              href={DeployTargetDocs[deployTarget]}
               target="_blank"
               rel="noreferrer"
             >
-              Vercel CLI
+              {humanizeString(deployTarget)}
             </StyledLink>{" "}
-            to publish your project on Vercel.
           </Text>
         </Grid>
+
+        <Select
+          fullWidth
+          css={{ zIndex: Number.MAX_SAFE_INTEGER.toString() }}
+          id={id}
+          value={deployTarget}
+          options={Object.keys(DeployTargets)}
+          getLabel={(value) => humanizeString(value as string)}
+          onChange={(value) =>
+            selectDeployTarget(value as keyof typeof DeployTargets)
```

Comment:
use type guard to check type example https://github.com/webstudio-is/webstudio-builder/blob/f5882256b8d542ca1f5b00e248d8e6a66b51e09f/packages/asset-uploader/src/utils/mime.ts#L40